<launch>
    <node pkg="get_2d_map" type="main" name="loam_to_2d" output="screen">
        <remap from="/loam_velodyne" to="/laser_cloud_surround" />
        <param name="resolution" type="double" value="0.02" />
        <param name="width" type="int" value="10" />
        <param name="height" type="int" value="10" />
        <param name="laser_frame" type="string" value="/base_link" />
    </node>
    <!-- <node pkg="pcl_ros" type="pointcloud_to_pcd" name="pointcloud_to_pcd">
        <remap from="input" to="/loam_3d_map" />
        <param name="frame_id" type="string" value="/map" />
        <param name="prefix" type="string" value="/home/zh/workspace/catkin_ws/src/bag_file/pcd/get_2d_map/map" />
        <param name="binary" type="bool" value="false" />
        <param name="compressed" type="bool" value="false" />
    </node> -->
</launch>